cephfs/path.go

```diff
@@ -12,14 +12,27 @@ import (
 	"unsafe"
 )
 
+func (mount *MountInfo) validateMount() error {
```

Comment:
Let's call this `validate` to match the other parts of the code.
We should also place this in cephfs.go to keep it closer to the defintion of the MountInfo type itself.

Reply:
Done